src/simulator/instruction.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg_attr(not(test), allow(dead_code))]
+
+extern crate foxbox_taxonomy;
+
+use foxbox_taxonomy::util::Id;
+use foxbox_taxonomy::values::*;
+use foxbox_taxonomy::devices::*;
+use foxbox_taxonomy::selector::*;
+use foxbox_taxonomy::api::{WatchEvent, WatchOptions, Error};
+
+#[derive(Serialize, Deserialize, Debug)]
+/// Instructions given to the simulator.
+pub enum Instruction {
+    AddNodes(Vec<Node>),
+    AddGetters(Vec<Channel<Getter>>),
+    AddSetters(Vec<Channel<Setter>>),
+    InjectGetterValue{id: Id<Getter>, value: Value},
```

Comment:
Apparently, we're supposed to add spaces, as follows:
`InjectGetterValue { id: ... Value },`